regression/linting_all_readmes.test.ts

```diff
@@ -0,0 +1,6 @@
+import { runLinter } from "./linting"
+import { allSpecs } from "./specsFilter"
+
+describe("validate all readme files", () => {
```

Comment:
9999 looks like magic number. Do you mean timeout?

Reply:
right